packages/react/docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+<details open="open">
+<summary><strong>TABLE OF CONTENTS</strong></summary>
+
+- [Development environment setup](#development-environment-setup)
+- [Issues and feature requests](#issues-and-feature-requests)
+  - [How to submit a Pull Request](#how-to-submit-a-pull-request)
+- [Publishing packages](#publishing-packages)
+  - [Publishing `@next` package versions](#publishing-next-package-versions)
+- [License](#license)
+
+</details>
+
+## Development environment setup
+
+The current monorepo setup is based on:
+
+- [Lerna](https://github.com/lerna/lerna) used mostly for task running and versioning.
+- [npm workspaces](https://docs.npmjs.com/cli/v8/using-npm/workspaces) used for package linking.
+
+### A note on commit messages
+
+The processes required for Lerna to manage releases and changelogs is done through the [conventional-commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
+
+To set up your development environment, please follow these steps:
+
+1. Clone the repo
+
+   ```sh
+   git clone https://github.com/clerkinc/javascript
+   ```
+
+2. Change into the directory and install dependencies
+
+   ```sh
+   cd javascript
+   npm install
+   ```
+
+3. Build all the packages in the monorepo by running:
+
+   ```sh
+   npm run build
+   ```
+## Issues and feature requests
+
+You've found a bug in the source code, a mistake in the documentation or maybe you'd like a new feature? You can help us by [submitting an issue on GitHub](https://github.com/clerkinc/javascript/issues). Before you create an issue, make sure to search the issue archive -- your issue may have already been addressed!
+
+Please try to create bug reports that are:
+
+- _Reproducible._ Include steps to reproduce the problem.
+- _Specific._ Include as much detail as possible: which version, what environment, etc.
+- _Unique._ Do not duplicate existing opened issues.
+- _Scoped to a Single Bug._ One bug per report.
+
+**Even better: Submit a pull request with a fix or new feature!**
+
+### How to submit a Pull Request
+
+1. Search our repository for open or closed
+   [Pull Requests](https://github.com/clerkinc/javascript/pulls)
+   that relate to your submission. You don't want to duplicate effort.
+2. Fork the project
+3. Create your feature branch (`git checkout -b feat/amazing_feature`)
+4. Commit your changes (`git commit -m 'feat: Add amazing_feature'`). Clerk uses [conventional commits](https://www.conventionalcommits.org), so please follow the specification in your commit messages.
+5. Push to the branch (`git push origin feat/amazing_feature`)
+6. [Open a Pull Request](https://github.com/clerkinc/javascript/compare?expand=1)
+
+## Publishing packages
```

Comment:
🙃 
IMO I would skip the next two sections and keep them only at the repo as is.

Reply:
We really want a consistent `CONTRIBUTING.md` file for all the open-source packages so they don't vary by package. Which sections are you saying to remove?